isis/src/docsys/build/homepage.xml

```diff
@@ -29,4 +29,10 @@
     Reconnaissance Orbiter, MESSENGER, Lunar Reconnaissance Orbiter,
     Chandrayaan, Dawn, Kaguya, and New Horizons.
   </p>
+  <h2><a name=">AstrogeologySupportingDocumentation">Astrogeology Supporting Documentation</a></h2>
+  <p>
+    Supporting ISIS Documentation containing tutorials, how-to guides and explanations of how to use the software 
+    has been moved to <a href="https://astrogeology.usgs.gov/docs/">https://astrogeology.usgs.gov/docs/</a>. Previous docs 
+    for older versions of ISIS can be accessed on the left navigation pane. 
```

Comment:
I feel there is a higher visibility way to do this?

Reply:
Something between [USWDS Cards](https://designsystem.digital.gov/components/card/) and the tiles on the new ISIS Applications by Category page?

Reply:
A bigger banner can also be a place for info/news like this

Reply:
<img width="1303" alt="Screenshot 2024-11-12 at 14 46 18" src="https://github.com/user-attachments/assets/a6d7a45e-0339-4f04-b947-56cc2eeb2d94">

Reply:
```html
<style>

    /* Should go to isisStyleCommons.css, replace current #isis-banner style */

    #isis-banner {
        background: 
        linear-gradient(
            rgba(0, 0, 0, 0.2), 
            rgba(0, 0, 0, 0.3)
            ),
        url(../img/moon-cr.jpg);
        color: white;
        padding: 4rem;
        background-size: cover;
        display: flex;
        flex-direction: row;
        flex-wrap: wrap;
        justify-content: space-between;
    }

    .news-card {
        margin: 18px 30px;
        padding: 30px;
        border: 3px solid darkslategray;
        border-radius: 5px;
        width: 28rem;
    }

    .news-card a {
        color: white;
    }

    /* Probably should add the usa-button style too */
</style>


<div id="isis-banner">  <!--replaces current id="isis-banner" div-->

    <div>
        <div class="isis-logo"></div>
        <p>
        <em>Integrated Software for Imagers and Spectrometers</em>
        </p>
    </div>
        
    <div class="news-card">
        <h2>
        Some Docs have Moved
        </h2>
        <p>
        Manuals for each ISIS application <a href="#TODO: link to Applications By Category">remain here</a>, but tutorials and guides have been moved to ASC Docs.
        </p>
        <button>
        Visit ASC Docs
        </button>
    </div>

</div>


```